README.md

```diff
@@ -41,12 +41,27 @@ The following example shows how to read in a FITACF file, one of the SuperDARN's
 
 ```python
 import pydarn
-dmap_file = "./20180410.C0.sas.fitacf"
-dmap_reader = pydarn.DarnRead(dmap_file)
-dmap_data = dmap_reader.read_fitacf() 
-
-# dmap_data[record number][paramter name].value
-print(dmap_data[0]['bmnum'].value) 
+import bz2
+
+#read in FITACF file that is compressed with bzip2
+fitacf_file = "20170514.C0.rkn.fitacf.bz2"
+with bz2.open(fitacf_file) as fp:
+        fitacf_stream = fp.read()
+	
+# Reading in the compression stream
+dmap = pydarn.DarnRead(fitacf_stream, True)
+fitacf_dmap = dmap.read_fitacf()
+
+#convert to a dictionary
+fitacf_data = pydarn.dmap2dict(fitacf_dmap)
+
+#Do some parsing of the data!
+elv_list = []
+for fitacf_rec in fitacf_data:
+    if fitacf_rec['bmnum'] == 5:
+        elv_list.append(fitacf_rec['elv'])
+
+print(elv_list[100])
```

Comment:
Is this a debugging statement?  If so putting a comment here to that effect would be useful.

Reply:
Ok... was a bit lost in the reviewer window.  It looks now like this is example code and you're printing the output for obvious reasons.  If correct, just mark this as resolved :-P